docs/api/sql/Function.md

```diff
@@ -561,6 +562,16 @@ The optional third parameter controls the buffer accuracy and style. Buffer accu
 - `mitre_limit=#.#` : mitre ratio limit and it only affects mitred join style. `miter_limit` is an accepted synonym for `mitre_limit`.
 - `side=both|left|right` : The option `left` or `right` enables a single-sided buffer operation on the geometry, with the buffered side aligned according to the direction of the line. This functionality is specific to LINESTRING geometry and has no impact on POINT or POLYGON geometries. By default, square end caps are applied.
 
+Mode of buffer calculation (Since: `v1.6.0`):
+
+- Planar Buffering (default): When `useSpheroid` is false, `ST_Buffer` performs standard planar buffering based on the provided parameters.
+- Spheroidal Buffering:
+    - When `useSpheroid` is set to true, the function returns the spheroidal buffer polygon for more accurate representation over the Earth.
+    - ST_Buffer first determines the most appropriate Spatial Reference Identifier (SRID) for a given geometry, based on its spatial extent and location, using `ST_BestSRID`.
```

Comment:
ST_BestSRID is only used when the input geom has no SRID or SRID == 4325. Please rephrase the sentences to make it clear.

In addition, what will happen to geometries that cross the dateline?